### ISSUE TITLE
Fix false positive on throw for CreateContainOneNewFactoryRule

### ISSUE THREAD

**Issue by dereuromark** [spryker/architecture-sniffer#20]
See
```
    public function createFormatter($formatType)
    {
        if (!$formatType) {
            $formatType = 'json';
        }
        switch ($formatType) {
            case 'json':
                return new JsonFormatter($this->getUtilEncoding());
        }

        throw new FormatterNotFoundException(sprintf('Formatter for type `%s` not found', $formatType));
    }
```

> ApiCommunicationFactory::createFormatter() contains 2 new statements which violates rule 'A create*() method must contain exactly 1 `new` statement.'

**Comment by dereuromark** [comment]
Fixed in https://github.com/spryker/architecture-sniffer/commit/6725dfab52743f5704549b4758fd2df3478b69ec
